controllers/v1/flowers/gardens_controller.go

```diff
@@ -0,0 +1,34 @@
+package flowers_controllers
+
+import (
+	"context"
+	"log"
+
+	mongo_connection "github.com/ice-cream-backend/database"
+	flowers_models "github.com/ice-cream-backend/models/v1/flowers"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func GetFlowerList() []flowers_models.Flowers {
```

Comment:
@yukari-n-erb I think the function name can just be `GetFlowers`. We know it will be a list of flowers from the name of the function.